api/graphql/schema.graphql

```diff
@@ -134,37 +158,41 @@ type Mutation {
     id: ID!
     username: String
     password: String
-    admin: Boolean
-  ): User! @isAdmin
+    roleId: ID
+    admin: Boolean @deprecated(reason: "The roles system replaces the old admin boolean.")
```

Comment:
I really like that you deprecated the old method. Then when we want to make a 3.0 release at some point, we can refactor out all the deprecations.